x-pack/plugins/infra/public/components/waffle/waffle_group_by_controls.tsx

```diff
@@ -12,32 +12,101 @@ import {
   EuiFilterGroup,
   EuiPopover,
 } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage, InjectedIntl, injectI18n } from '@kbn/i18n/react';
 import React from 'react';
 import { InfraNodeType, InfraPathInput, InfraPathType } from '../../../common/graphql/types';
 
 interface Props {
   nodeType: InfraNodeType;
   groupBy: InfraPathInput[];
   onChange: (groupBy: InfraPathInput[]) => void;
+  intl: InjectedIntl;
 }
 
 const OPTIONS = {
   [InfraNodeType.pod]: [
-    { text: 'Namespace', type: InfraPathType.terms, field: 'kubernetes.namespace' },
-    { text: 'Node', type: InfraPathType.terms, field: 'kubernetes.node.name' },
+    {
+      text: i18n.translate('xpack.infra.waffleGroupByControls.optionsNamespaceText', {
```

Comment:
I believe it's better:
```suggestion
      text: i18n.translate('xpack.infra.waffle.groupByOptions.namespaceLabel', {
```
The same and for labels in `OPTIONS` object.

@pavel06081991 what do you think?